src/Gui/GestureNavigationStyle.cpp

```diff
@@ -101,13 +100,13 @@ const char* GestureNavigationStyle::mouseButtons(ViewerMode mode)
 {
     switch (mode) {
     case NavigationStyle::SELECTION:
-        return QT_TR_NOOP("Tap. Or click left mouse button.");
+        return QT_TR_NOOP("Tap OR click left mouse button.");
     case NavigationStyle::PANNING:
-        return QT_TR_NOOP("Drag screen with two fingers. Or press right mouse button.");
+        return QT_TR_NOOP("Drag screen with two fingers OR press right mouse button.");
     case NavigationStyle::DRAGGING:
-        return QT_TR_NOOP("Drag the screen with one finger. Or press left mouse button. In Sketcher and other edit modes, hold Alt in addition.");
+        return QT_TR_NOOP("Drag screen with one finger OR press left mouse button. In Sketcher and other edit modes, hold Alt in addition.");
     case NavigationStyle::ZOOMING:
-        return QT_TR_NOOP("Pinch (put two fingers on the screen and drag them apart/to each other). Or scroll middle mouse button. Or PgUp/PgDown on keyboard.");
+        return QT_TR_NOOP("Pinch (place two fingers on the screen and drag them apart from or towards each other) OR scroll middle mouse button OR PgUp/PgDown on keyboard.");
```

Comment:
@DeepSOIC please review potential changes. This and everything above this within GestureNavigationStyle.cpp

Reply:
looks good to me =)